src/convertToObject.js

```diff
@@ -3,16 +3,26 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in [stylesString.js]
+ * (./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';').map(value =>
+    value.split(':')).map(el => el.map(subel => subel.trim()))
+    .filter(value => value.length > 1)
+    .reduce((object, [key, value]) => {
```

Comment:
Better not use this name
https://mateacademy.slack.com/archives/C02T596SHUZ/p1644845021054899